cherryontech/jupiter-jumpers
# Intro tooltip

## Description

1. Adds a tooltip that points to the "Add Log" button when a user first logs in.


## Related Story Card

_Provide a link to the relevant story card._

## Type of Changes

Feature

## Acceptance Criteria


## Update Screenshots

### After
<img width="1429" alt="Screenshot 2024-06-10 at 7 55 07 PM" src="https://github.com/cherryontech/Uncloud/assets/47403135/94e772ac-1ed8-413d-a0e7-344f67795026">
<img width="1441" alt="Screenshot 2024-06-10 at 7 55 19 PM" src="https://github.com/cherryontech/Uncloud/assets/47403135/ca56403e-2ccb-43fb-aeca-4b5db0606be6">

## Testing Instructions

1. Create a new account
2. Press "Get Started" in the confirmation modal to close it
3. Ensure the tooltip shows up pointing to "Add a log", with a shake effect. 
4. Click anywhere (except "Got it") -- the tooltip should close. 
5. Refresh the page and ensure it doesn't appear again.
6. Repeat steps 1-3
7. Click "Got it" -- the tooltip should close.

## Learnings (Optional)
